main.go

```diff
@@ -261,6 +265,74 @@ func writeNewXWithOptionsAndDefaults(buf *jen.File, c Config) {
 	})
 }
 
+var (
+	DebugMapFieldTag = "debugmap"
+	SensitiveNames   = []string{"secure"}
+)
+
+func writeDebugMap(buf *jen.File, st *types.Struct, c Config) {
+	newFuncName := fmt.Sprintf("DebugMap")
```

Comment:
What do you think about doing this as [GoString](https://pkg.go.dev/fmt#GoStringer)? 

That should make it a lot harder to accidentally print sensitive fields